mmocr/utils/ocr.py

```diff
@@ -166,10 +167,10 @@ def det_and_recog_inference(args, det_model, recog_model):
             box_res['box_score'] = float(bbox[-1])
             box = bbox[:8]
             if len(bbox) > 9:
-                min_x = min(bbox[0:-1:2])
-                min_y = min(bbox[1:-1:2])
-                max_x = max(bbox[0:-1:2])
-                max_y = max(bbox[1:-1:2])
```

Comment:
Are you sure that this is an indexing bug ? By removing the `-1` you are including the box score in one of the new variables which should only include the bounding shape coordinates. 

I might add that this whole part should be reworked because it does not make good use of the arbitrary shaped bounding regions returned by some detection models. Something should be created to pre-process those coordinates with a combination of automated warping and perspective projections to return the canonical form of the text. This would make recognition much more reliable on curved and inclined text.